### PR TITLE
Add a page_details partial to render annotation text.

### LINKED ISSUE #842: Implement page details section "Page detail" show page
Design in #727

### PR DESCRIPTION
**Pull request by jkeck**
Closes #842 

### Canvas: canvas-dddcd264a38dea30849f17da8ddb59c2
fh878gz0315-canvas-dddcd264a38dea30849f17da8ddb59c2

### CODE REVIEW

**Comment by jkeck** [comment]
See 217d1af9c5917c4014feef854547bb50b5955f54 for the most relevant commit.  The other two are small UI refactors necessary to get things working.

File: spec/features/canvas_display_spec.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'Canvas Displaly', type: :feature do
+  let(:exhibit) { FactoryGirl.create(:exhibit, slug: 'default-exhibit') }
```
Comment (camillevilla) [inline spec/features/canvas_display_spec.rb]: @jkeck can you remove `FactoryGirl`? This `create` should still work, and it'll mean less work in #852.

File: spec/features/canvas_display_spec.rb [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'Canvas Displaly', type: :feature do
```
Comment (camillevilla) [inline spec/features/canvas_display_spec.rb]: just a lil typo here ^

**Comment by jkeck** [comment]
Good catch @camillevilla!  I'm pushed up the fixes.

**Pull request merged**